### PR TITLE
More overall Documentation tweaking, and updating README to 19.2

### PR DESCRIPTION
**Pull request by MounaSafiHarab**
Mostly Readme version change, but also more documentation changes as I read them all at once...

### CODE REVIEW

**Comment by MounaSafiHarab** [comment]
@cmadjar @nicolasbrossard @christinerogers @davidblader 

I am starting to read the documentation as a whole, so I will be updating this PR (sending more commits) as I read more.
Please feel free to also read and either send separate PRs or push to this branch.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
-This Readme covers release 19.0 of the LORIS Imaging Insertion Pipeline for Ubuntu or CentOS systems
+This Readme covers release 19.2 of the LORIS Imaging Insertion Pipeline for Ubuntu or CentOS systems
 
-This repo accompanies the [LORIS neuroimaging data platform main repo](https://github.com/aces/Loris/releases)</b>, release 19.0.*.<br>
+This repo accompanies the [LORIS neuroimaging data platform main repo](https://github.com/aces/Loris/releases)</b>, release 19.2.*.<br>
```
Comment (MounaSafiHarab) [inline README.md]: on the line below, we should point to our documentation instead of the wiki, don't you think @cmadjar?

Comment (cmadjar) [reply README.md]: @MounaSafiHarab I totally agree :)

**Comment by christinerogers** [comment]
Hi @MounaSafiHarab, Readme looks great! One or two minor comments for now: 
- update the imaging_install.sh Line number that should point to chown commands to run/verify
- note about the Readme ->  maybe we should specify "for your installed version" ? given  recent branching update timelines etc...

File: docs/01-Introduction.md [hunk 2]
```diff
@@ -1,50 +1,55 @@
...
 please consult the [LORIS wiki][1].
 
 LORIS-MRI is a set of libraries responsible for the insertion,
-organization, and archival of uploaded imaging datasets.
- It expects an uploaded, compressed file containing
-a [DICOM][2] study composed of several DICOM files. These DICOM files
- will be archived on the server and converted to [MINC][3] and (optionally)
+organization, and archival of uploaded imaging datasets. 
+It expects an uploaded, compressed file containing 
+a [DICOM][2] study composed of several DICOM files. These DICOM files
```
Comment (christinerogers) [inline docs/01-Introduction.md]: DICOM study -> DICOM scan session

File: docs/01-Introduction.md [hunk 3]
```diff
@@ -1,50 +1,55 @@
...
 These documents assume you have some knowledge regarding LORIS and
 a functioning installation. For information regarding LORIS itself,
 please consult the [LORIS wiki][1].
 
 LORIS-MRI is a set of libraries responsible for the insertion,
-organization, and archival of uploaded imaging datasets.
- It expects an uploaded, compressed file containing
-a [DICOM][2] study composed of several DICOM files. These DICOM files
- will be archived on the server and converted to [MINC][3] and (optionally)
+organization, and archival of uploaded imaging datasets.
```
Comment (christinerogers) [inline docs/01-Introduction.md]: archival -> archiving 
(archival is much more common as an adjective so the sentences reads a tiny bit funny)

File: docs/01-Introduction.md [hunk 4]
```diff
@@ -1,50 +1,55 @@
...
 [NIfTI][4] files. Knowledge of these file formats can be helpful, but are not 
 necessary for using or installing LORIS-MRI.
 
 LORIS-MRI allows you to easily organize and archive your imaging datasets
-and links them with
- corresponding behavioral data in LORIS. Scans can be viewed and quality
- controlled through the user's web browser, facilitating collaboration
- between radiologists, clinicians and researchers.
+and links them with corresponding behavioral data in LORIS. Scans can be viewed 
+and quality controlled through the user's web browser, facilitating
```
Comment (christinerogers) [inline docs/01-Introduction.md]: through the user's web browser-> "in the LORIS front end via web browser"

File: docs/01-Introduction.md [hunk 5]
```diff
@@ -1,50 +1,55 @@
...
+![user_story](images/user_story.png)
 
-### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
 LORIS-MRI allows multiple ways to upload scans, but typically, users
-upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging
-Uploader.  LORIS requires that the uploaded file name follow the naming
-convention PSCID_CANDID_VISIT-LABEL. 
+upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder via the Imaging
+Uploader module. LORIS requires that the uploaded file name follow the naming
+convention `PSCID_CANDID_VISIT-LABEL`.
```
Comment (christinerogers) [inline docs/01-Introduction.md]: I'd remove the hyphen in visit label  (sorry, truly tiny comment)

File: docs/01-Introduction.md [hunk 6]
```diff
@@ -1,50 +1,55 @@
...
-convention PSCID_CANDID_VISIT-LABEL. 
+upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder via the Imaging
+Uploader module. LORIS requires that the uploaded file name follow the naming
+convention `PSCID_CANDID_VISIT-LABEL`. 
 In addition, all DICOM datasets uploaded via the Imaging Uploader or 
 transferred on the LORIS-MRI server must be free of any identifying 
-information (e.g. patient name). A tool can be provided to the sites to 
-facilitate deidentification. Please contact the LORIS team for details.
+information (*e.g.* patient name). A tool can be provided to the sites to 
+facilitate de-identification. Please contact the LORIS team for details.
```
Comment (christinerogers) [inline docs/01-Introduction.md]: Since DICAT is open-source, I'd just point them to it.  (if there's any reason why not, feel free to ignore this comment)

File: docs/01-Introduction.md [hunk 7]
```diff
@@ -1,50 +1,55 @@
...
 
 The LORIS-MRI pipeline starts once the scans are uploaded to the server.
```
Comment (christinerogers) [inline docs/01-Introduction.md]: "starts...uploaded" -> "is designed to run on scans once they are uploaded"

File: docs/01-Introduction.md [hunk 8]
```diff
@@ -1,50 +1,55 @@
...
 The pipeline can start automatically if the autolaunch configuration is
```
Comment (christinerogers) [inline docs/01-Introduction.md]: names this exact setting, mention "via the Configuration module"

File: docs/01-Introduction.md [hunk 9]
```diff
@@ -1,50 +1,55 @@
...
 set, otherwise manual intervention is required by a developer with backend
```
Comment (christinerogers) [inline docs/01-Introduction.md]: "a back-end administrator can manually run the pipeline. "
(because not necessarily a developer who needs to know coding)

File: docs/01-Introduction.md (hunk 2 above)
Comment (christinerogers) [inline docs/01-Introduction.md]: DICOM study composed of several DICOM files -> DICOM scan session composed of many DICOM slice files.

File: docs/01-Introduction.md [hunk 10]
```diff
@@ -1,50 +1,55 @@
...
-information (e.g. patient name). A tool can be provided to the sites to 
-facilitate deidentification. Please contact the LORIS team for details.
+information (*e.g.* patient name). A tool can be provided to the sites to 
+facilitate de-identification. Please contact the LORIS team for details.
 
 The LORIS-MRI pipeline starts once the scans are uploaded to the server.
 The pipeline can start automatically if the autolaunch configuration is
 set, otherwise manual intervention is required by a developer with backend 
 access to the server. More details about these options and the scripts
 needed to run the pipeline can be found [here](05-PipelineLaunchOptions.md).
```
Comment (christinerogers) [inline docs/01-Introduction.md]: "These options and scripts are detailed in the [Pipeline Options documentation]"

File: docs/01-Introduction.md [hunk 11]
```diff
@@ -1,50 +1,55 @@
...
-Imaging Uploader, where descriptive messages can be consulted.
+Insertion progress can be tracked by the user through the Log Viewer in the
+Imaging Uploader module, where descriptive messages can be consulted.
 The output of the main key steps in the insertion progress can also be consulted 
-through the LORIS DICOM archive module for successfully archived DICOM datasets, 
-Imaging Browser for MINC-generated files that pass the study-defined MRI protocol, 
-and 3D or 4D navigation of these MINC files in BrainBrowser. More details on the
-capabilities of BrainBrowser can be found [here.][5]
+through:
+ - the LORIS DICOM archive module for successfully archived DICOM datasets
```
Comment (christinerogers) [inline docs/01-Introduction.md]: module name: A in Archive

File: docs/01-Introduction.md [hunk 12]
```diff
@@ -1,50 +1,55 @@
...
+Insertion progress can be tracked by the user through the Log Viewer in the
+Imaging Uploader module, where descriptive messages can be consulted.
 The output of the main key steps in the insertion progress can also be consulted 
-through the LORIS DICOM archive module for successfully archived DICOM datasets, 
-Imaging Browser for MINC-generated files that pass the study-defined MRI protocol, 
-and 3D or 4D navigation of these MINC files in BrainBrowser. More details on the
-capabilities of BrainBrowser can be found [here.][5]
+through:
+ - the LORIS DICOM archive module for successfully archived DICOM datasets 
+ - the Imaging Browser module for MINC-generated files that pass the
```
Comment (christinerogers) [inline docs/01-Introduction.md]: MINC-generated -> MINC files (generated from DICOM)

File: docs/01-Introduction.md (hunk 2 above)
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: done

**Comment by MounaSafiHarab** [comment]
> Hi @MounaSafiHarab, Readme looks great! One or two minor comments for now:
> 
> update the imaging_install.sh Line number that should point to chown commands to run/verify
> note about the Readme -> maybe we should specify "for your installed version" ? given recent branching update timelines etc...

Good point re. point 2. Fixed.
But for point 1., I think line 79 is still valid as the comment is referring to `mkdir/chown/...` and the `mkdir` part starts at line 79.

File: docs/01-Introduction.md [hunk 13]
```diff
@@ -1,50 +1,55 @@
 # 1.0 - Introduction
 
 ### 1.1 What is LORIS-MRI? 
-LORIS-MRI is the backbone of the imaging component that makes up LORIS.
- It is maintained in a separate repository so that it can be installed
- on the file server and separated from the web server.
+
+LORIS-MRI is the backbone of the imaging component that makes up LORIS.
```
[the remark this answers is not in the document]
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: done, with the only exception that I changed the word 'pre-processing' to 'inserting'.

File: docs/01-Introduction.md [hunk 14]
```diff
@@ -1,50 +1,55 @@
...
 ### 1.1 What is LORIS-MRI? 
-LORIS-MRI is the backbone of the imaging component that makes up LORIS.
- It is maintained in a separate repository so that it can be installed
- on the file server and separated from the web server.
+
+LORIS-MRI is the backbone of the imaging component that makes up LORIS. 
+It is maintained in a separate repository so that it can be installed 
+on the file server and separated from the web server.
 These documents assume you have some knowledge regarding LORIS and
 a functioning installation. For information regarding LORIS itself,
```
[the remark this answers is not in the document]
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: ok

File: docs/01-Introduction.md (hunk 9 above)
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: ok

File: docs/01-Introduction.md (hunk 2 above)
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: I don;t mind putting DICOM `slice` files. But are we certain that it is only slice files?
@cmadjar ?

File: docs/01-Introduction.md (hunk 10 above)
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: ok

File: docs/01-Introduction.md (hunk 11 above)
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: ok

File: docs/01-Introduction.md (hunk 2 above)
Comment (cmadjar) [reply docs/01-Introduction.md]: Actually, DICOM files could also be volumes not just slice so I would not change the wording here.

Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: Okay, so based on the comments I now have:
`a [DICOM][2] scan session composed of many DICOM files.`

File: README.md [hunk 15]
```diff
@@ -77,7 +77,7 @@ The installer will also set Apache group ownership of certain /data/ subdirector
 **INSTALLATION COMPLETE!**
 
 Please refer to the [Install](docs/02-Install.md) section in the 
-[LORIS-MRI documentation](docs/) for:
+[LORIS-MRI documentation](docs/) for your installed version for:
```
Comment (cmadjar) [inline README.md]: **of** your installed version for
 maybe?

File: docs/01-Introduction.md (hunk 5 above)
Comment (um4r12) [reply docs/01-Introduction.md]: Would it be helpful to also include an example? `PSCID_CANDID_VISITLABEL` (_ex. MON001_987664_V01_)

File: docs/01-Introduction.md [hunk 16]
```diff
@@ -1,50 +1,56 @@
...
+ 
+![user_story](images/user_story.png)
 
-### 1.2 How does LORIS-MRI work?  ![user_story](images/user_story.png)
 LORIS-MRI allows multiple ways to upload scans, but typically, users
-upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder to Imaging
-Uploader.  LORIS requires that the uploaded file name follow the naming
-convention PSCID_CANDID_VISIT-LABEL. 
+upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder via the Imaging
+Uploader module. LORIS requires that the uploaded file name follow the naming
```
Comment (um4r12) [inline docs/01-Introduction.md]: Include something along the lines: "... users upload a compressed (.tgz, .tar.gz, or .zip) DICOM folder via the Imaging Uploader module that is only composed of DCM files." 
I know the `__MACOSX` folder is removed, but insertion scripts fail if other files are included. Ex. Datasets directly uploaded from Philip scanners contain files & and exe files that halt the pipeline, and cause the `Progress` column to remain as `In Progress` indefinitely, until manually removed.

File: docs/02-Install.md [hunk 17]
```diff
@@ -108,22 +108,23 @@ the [LORIS repository: Imaging Uploader Specification](https://github.com/aces/L
...
 
 
 3. **Imaging Browser**
 
-Imaging Browser accesses the PIC images directly from the filesystem where they
-are stored. It also provides the option to doownload some files.  Ensure that:
-- `/data/$PROJ` directory and subdirectories are readable and executable by
+The Imaging Browser module accesses the screenshot (PIC) images directly from 
+the filesystem where they are stored. It also provides the option to download 
+some files. Ensure that:
```
Comment (um4r12) [inline docs/02-Install.md]: I'm not sure if this is still relevant, but: "It also provides the option to download some files." => "By default, it also provides the option to download individual scans as a MINC file, or the entire session as a compressed file (_.tar_ extension)."

File: docs/02-Install.md [hunk 18]
```diff
@@ -153,11 +154,11 @@ Data loaded in this module gets populated automatically by the insertion
 scripts. As such, scans whose parameters can't be matched against the 
 `mri_protocol` table during the imaging insertion process, will be flagged as 
 protocol violations and will not have their MINC/NIfTI volumes loaded in the 
-database. The type of error (scan identification, protocol violation) will be 
-listed and can be reviewed from the front-end.
+Imaging Browser module. The type of error (scan identification, protocol 
+violation) will be listed and can be reviewed from the front-end.
```
Comment (um4r12) [inline docs/02-Install.md]: Modify to include: "Violated scans can be viewed and the type of error (scan identification, protocol violation) can be reviewed from the front-end."

File: docs/02-Install.md [hunk 19]
```diff
@@ -167,29 +168,29 @@ More detailed specifications can be consulted in the
 - `/data/*` subdirectories were created by the imaging install script. If not,
     it may be due to `root:root` ownership of the `/data/` mount on your
     system. Ensure these subdirectories are created manually, particularly:
-    `/data/$PROJ/data/*`, `/data/$PROJ/bin/mri/` and `/data/incoming/`
+    `/data/$PROJECT/data/*`, `/data/$PROJECT/bin/mri/` and `/data/incoming/`
```
Comment (um4r12) [inline docs/02-Install.md]: Do you think we should explicitly name the `tarchive`, `pic`, `logs`, `trashbin`, `batch_output`, `jiv`, `assembly`  subdirectories instead of just `/data/$PROJECT/data/*`?

File: docs/03-TechnicalInfrastructure.md [hunk 20]
```diff
@@ -4,13 +4,13 @@
...
 
 ```
 ## Imaging pipeline file directory structure
 /
 |__ data
-    |__ project
+    |__ $PROJECT
         |__ bin
         |   |__ mri
         |__ data
```
Comment (um4r12) [inline docs/03-TechnicalInfrastructure.md]: I'm not sure if this is accurate. For my sandbox, and our own project, the `incoming` directory is not nested within `/data/$PROJECT/data` but rather just under `/data/`. 
When using `imaging_install.sh`, by default, the `pipelines`, `protocols` subdirectories are not created under `/data/$PROJECT/data/` either. Maybe we should add an `optional` tag or something?

File: docs/03-TechnicalInfrastructure.md [hunk 21]
```diff
@@ -90,10 +90,10 @@ Jiv images produced by the imaging insertion pipeline are organized per
...
 #### The `logs` directory
 
-The logs of the scripts are created under `data/logs` in `/data/project`.
+The logs of the scripts are created under `data/logs` in `/data/$PROJECT`.
     
 ```
-## Content of the /data/project/data/logs directory
+## Content of the /data/$PROJECT/data/logs directory
 .
 |__ TarLoad.log
```
Comment (um4r12) [inline docs/03-TechnicalInfrastructure.md]: Is the `TarLoad.log` a directory within `/data/$PROJECT/data/logs/`? If so, my `/data/$PROJECT/data/logs` is not structured as such. Currently, I just have the `TarLoad-X-XX-XXXXXX.log` files present. I understand that the other folders are created when other pipelines are executed, but should be outdented. Ex.
```
## Content of the /data/$PROJECT/data/logs directory
.
|__ TarLoad.log
|__ DTIPrep_pipeline
|   |__ DTI_QC`date`.log
|   |__ DTI_QC`date`.log
|__ DTIPrep_register
|   |__ DTIregister`date`.log
|   |__ DTIregister`date`.log
|__ registerProcessed
    |__ registerProcessed`date`.log
    |__ registerProcessed`date`.log
```

File: docs/01-Introduction.md (hunk 16 above)
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: good point; added the extra clarification!
but out of curiosity, can you show me when it halts? I have Philips scanners and they do indeed include some `.exe` files sometimes, but the pipeline stops at the very first step saying the uploads contains `x` number of files which are not of type DICOM...

File: docs/02-Install.md (hunk 17 above)
Comment (MounaSafiHarab) [reply docs/02-Install.md]: I agree with you it is useful for the user, but the advantage of saying 'some files' is that we link to Imaging Browser where this detailed info is kept and more importantly, maintained up to date (as an example, Imaging browser should/will also allow to download niftii and other types of files, so makes sense to add this info in Imaging Browser when the feature is changed and not worry about tracking it here).

@um4r12 
do you agree or you still strongly feel we should add it here?

File: docs/02-Install.md (hunk 18 above)
Comment (MounaSafiHarab) [reply docs/02-Install.md]: done

File: docs/02-Install.md (hunk 19 above)
Comment (MounaSafiHarab) [reply docs/02-Install.md]: done!

File: docs/03-TechnicalInfrastructure.md (hunk 20 above)
Comment (MounaSafiHarab) [reply docs/03-TechnicalInfrastructure.md]: agree on both points. Corrected now and along with your suggested correction, I reordered the directories to be in synch with the order the installer does them, and I added the missing `batch_output` one.

File: docs/03-TechnicalInfrastructure.md (hunk 21 above)
Comment (MounaSafiHarab) [reply docs/03-TechnicalInfrastructure.md]: agreed. I made the changes but I am roping @cmadjar here so she can undo and/or add changes on top of my changes.

File: docs/03-TechnicalInfrastructure.md [hunk 22]
```diff
@@ -90,31 +94,35 @@ Jiv images produced by the imaging insertion pipeline are organized per
...
-    |   |__ DTI_QC`date`.log
-    |__ DTIPrep_register
-    |   |__ DTIregister`date`.log
-    |   |__ DTIregister`date`.log
-    |__ registerProcessed
-        |__ registerProcessed`date`.log
-        |__ registerProcessed`date`.log
+|__ TarLoad-`hh-mm-xxxxxx`.log
+|__ DTIPrep_pipeline*
+|   |__ DTI_QC`date`.log
```
Comment (MounaSafiHarab) [inline docs/03-TechnicalInfrastructure.md]: @cmadjar 
the `date` thing is not being formatted properly because it is already in a code section!
not sure if it worth investigating this, but I can not seem to figure out how!

File: docs/02-Install.md (hunk 17 above)
Comment (um4r12) [reply docs/02-Install.md]: @MounaSafiHarab You're right, don't think it's necessary to add it here, will just make it difficult to track changes in different locations. I had been reading from top to bottom, and I think it is mentioned somewhere further down anyways.

**Comment by um4r12** [comment]
@MounaSafiHarab Looks good to me!

File: docs/01-Introduction.md [hunk 23]
```diff
@@ -1,51 +1,62 @@
...
- corresponding behavioral data in LORIS. Scans can be viewed and quality
- controlled through the user's web browser, facilitating collaboration
- between radiologists, clinicians and researchers.
+and links them with corresponding behavioral data in LORIS. Scans can be viewed 
+and quality controlled in the LORIS front end via web browser, facilitating 
+collaboration between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work? 
+ 
+![user_story](images/user_story.png)
```
Comment (um4r12) [inline docs/01-Introduction.md]: @davidblader do you still have the original figure? Can we make a slight modification to this? If the `DicomArchive` fails, it also changes the status to `Failure`. Would you be able to add an additional arrow pointing from the `Dicom Archive` to the `Upgrade Progress of entry in Imaging Uploader to 'Failure'`?

File: docs/03-TechnicalInfrastructure.md [hunk 24]
```diff
@@ -4,39 +4,43 @@
...
+            |__ tarchive
+            |__ pic
             |__ jiv
             |__ logs
-            |__ pic
-            |__ pipelines
-            |__ protocols
-            |__ tarchive
-            |__ trashbin
+            |__ assembly
```
Comment (cmadjar) [inline docs/03-TechnicalInfrastructure.md]: The directories under the data folder should be in alphabetical order I think. Not sure why this is not the case anymore. When you do `ls` it usually lists in alphabetical order. Like the * added to the `pipelines` and `protocol` folder though!

File: docs/03-TechnicalInfrastructure.md [hunk 25]
```diff
@@ -90,31 +94,35 @@ Jiv images produced by the imaging insertion pipeline are organized per
...
+|   |__ DTI_QC`date`.log
+|__ DTIPrep_register*
+|   |__ DTIregister`date`.log
+|   |__ DTIregister`date`.log
+|__ registerProcessed*
+    |__ registerProcessed`date`.log
+    |__ registerProcessed`date`.log
 ```
 
+`*` _denotes optional directories that are not automatically created by the
```
Comment (cmadjar) [inline docs/03-TechnicalInfrastructure.md]: I think they are automatically created by the`DTIPrep_pipeline.pl, `DTIPrepRegister.pl` and  `register_processed_data.pl` scripts.

**Comment by cmadjar** [comment]
@MounaSafiHarab FYI: once this is merged, I'll update the documentation for the localizer PR (that would avoid conflicts ;))

File: docs/03-TechnicalInfrastructure.md (hunk 24 above)
Comment (MounaSafiHarab) [reply docs/03-TechnicalInfrastructure.md]: I changed them to be the same order as in the installer script so we do not forget any directory;
but I will revert back to alphabetical order

File: docs/02-Install.md [hunk 26]
```diff
@@ -167,29 +168,33 @@ More detailed specifications can be consulted in the
 - `/data/*` subdirectories were created by the imaging install script. If not,
     it may be due to `root:root` ownership of the `/data/` mount on your
     system. Ensure these subdirectories are created manually, particularly:
-    `/data/$PROJ/data/*`, `/data/$PROJ/bin/mri/` and `/data/incoming/`
+    `/data/$PROJECT/bin/mri/`, `/data/incoming/`, and those inside 
+    `/data/$PROJECT/data/`, namely `trashbin`, `tarchive`, `pic`, `logs`, 
+    `jiv`, `assembly`, and `batch_output`.
```
Comment (MounaSafiHarab) [inline docs/02-Install.md]: @cmadjar 
changed the order here too as per your earlier comment on alphabetical order

File: docs/03-TechnicalInfrastructure.md (hunk 25 above)
Comment (MounaSafiHarab) [reply docs/03-TechnicalInfrastructure.md]: based on the two scripts you mention, 
I can only see the script creating a `logs/DTIPrep_pipeline/` and `/logs/DTIPrep_register/` using `mkdir -p -m` which creates the directory, and another script is creating the `protocols/` directory as follows:
```my $tooldir = $data_dir . "/protocols/" . $tool;
    `mkdir -m 770 $tooldir`    unless (-e $tooldir)```
But it only has `-m` option, no `-p` option, so am not sure anymore...
I updated the documentation based on what I see in the scripts, but feel free to give me specific instrustioins

**Comment by MounaSafiHarab** [comment]
@cmadjar 
check last commit for recent changes and your feedback.
I am not sure I got the DTIPrep part correct, I looked into the code and this is what I found. Please confirm!

File: docs/02-Install.md [hunk 27]
```diff
@@ -169,9 +169,8 @@ More detailed specifications can be consulted in the
     it may be due to `root:root` ownership of the `/data/` mount on your
     system. Ensure these subdirectories are created manually, particularly:
     `/data/$PROJECT/bin/mri/`, `/data/incoming/`, and those inside 
-    `/data/$PROJECT/data/`, namely `trashbin`, `tarchive`, `pic`, `logs`, 
-    `jiv`, `assembly`, and `batch_output`.
-
+    `/data/$PROJECT/data/`, namely `assembly`, `batch_output`, `jiv`, `logs`,
+    `pic`, `tarchive`, `trashbin`, and `tarchive`.
```
Comment (cmadjar) [inline docs/02-Install.md]: there is two tarchive here. Maybe remove the last one.

File: docs/03-TechnicalInfrastructure.md [hunk 28]
```diff
@@ -15,19 +15,20 @@ The root directory of the imaging part of a LORIS instance is typically
...
             |__ batch_output
-            |__ pipelines*
+            |__ jiv
+            |__ logs
+                |__ DTIPrep_pipeline*
+                |__ DTIPrep_register*
+            |__ pic
+            |__ tarchive
+            |__ trashbin
             |__ protocols*
```
Comment (cmadjar) [inline docs/03-TechnicalInfrastructure.md]: missing `pipelines` in the tree

Comment (MounaSafiHarab) [reply docs/03-TechnicalInfrastructure.md]: okay re-added it. But I removed it intentionally because the code never makes one, so I am not sure where it comes from?

File: docs/02-Install.md (hunk 27 above)
Comment (MounaSafiHarab) [reply docs/02-Install.md]: true, removed duplicate.

File: docs/01-Introduction.md (hunk 23 above)
Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: @davidblader 
ping ping...

**Comment by MounaSafiHarab** [comment]
> @MounaSafiHarab thanks for the changes! Two minor things to correct and should be good to go.

already addressed, with a question too... let me know...

File: docs/01-Introduction.md [hunk 29]
```diff
@@ -1,51 +1,62 @@
...
-and 3D or 4D navigation of these MINC files in BrainBrowser. More details on the
-capabilities of BrainBrowser can be found [here.][5]
+through:
+ - the LORIS DICOM Archive module for successfully archived DICOM datasets`*` 
+ - the Imaging Browser module for MINC files (generated from DICOM) that pass the 
+study-defined MRI protocol 
+ - BrainBrowser using 3D or 4D navigation of these MINC files. More details on 
+BrainBrowser's capabilities can be found [here.][5]
 
+`*` Please note that while acquisitions with `localizer` as their series description
```
Comment (MounaSafiHarab) [inline docs/01-Introduction.md]: @cmadjar 
this is where your PR #319 info conflict with the addition here.

File: docs/02-Install.md [hunk 30]
```diff
@@ -32,61 +32,61 @@ If age is not a critical factor in study visit scheduling, define Min value as
...
  * `Path to Tarchives`: Directory where the original DICOMs are archived; typically `/data/$PROJECT/data/tarchive/`
  * `Upload creation of candidates`: Enable or disable candidate creation into LORIS when running the insertion pipeline
  * `Project batch management used`: Enable or disable batch management
- * `If site is used`: Obsolete **To be confirmed**. This option used to allow the scans' incoming and archival paths to be configured based on the scanning sites
+ * `If site is used`: Obsolete **To be removed in the next release**. This  option used to allow the scans' incoming and archival paths to be configured based on the scanning sites
  * `Number of volumes in native DTI acquisitions`: Used by the DTIPrep pipeline
  * `Scan type of native T1 acquisition`: Name as specified in the `mri_scan_type` table. Used by the DTIPrep pipeline
- * `Max number of DTI rejected directions for passing QC`: Number of directions that can be rejected and still pass QC. Used by the DTIPrep pipeline
+ * `Max number of DTI rejected directions for passing QC`: Maximum number of directions that can be removed from a DTI scan to pass QC. Used by the DTIPrep pipeline
```
Comment (MounaSafiHarab) [inline docs/02-Install.md]: @cmadjar 
at the end of this section, we need to incorporate your new setting due to PR #319

File: docs/01-Introduction.md (hunk 29 above)
Comment (cmadjar) [reply docs/01-Introduction.md]: @MounaSafiHarab Thank you! I will edit this and try to push to this branch.

Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: cool, so I will merge now PR #319 then.

File: docs/01-Introduction.md (hunk 23 above)
Comment (davidblader) [reply docs/01-Introduction.md]: done

**Comment by cmadjar** [comment]
@MounaSafiHarab I modified the text to reflect the new localizer situation. See https://github.com/aces/Loris-MRI/pull/312/commits/bfa3cfe268eeee8313df65a09ea90fbdaa5a9cd1 to see the changes

File: docs/01-Introduction.md [hunk 31]
```diff
@@ -53,10 +53,12 @@ study-defined MRI protocol
  - BrainBrowser using 3D or 4D navigation of these MINC files. More details on 
 BrainBrowser's capabilities can be found [here.][5]
 
-`*` Please note that while acquisitions with `localizer` as their series description 
-are included in the DICOM archival step, they are excluded in all the steps that start 
-at, and follow the DICOM to MINC conversion. In a future release, the list of series 
-description to exclude will become customizable by projects.  
+`*` Please note that all acquisitions are included in the DICOM archival
+step (even the ones excluded at a further step of the pipeline). Specific
+acquisitions (such as localizers or scouts) can be excluded from the steps of
```
Comment (MounaSafiHarab) [inline docs/01-Introduction.md]: Code format for localizers and scouts

Comment (MounaSafiHarab) [reply docs/01-Introduction.md]: 1) I would rephrase here to say: "Please note that all acquisitions are included in the DICOM archival step. However, specific acquisitions ...".

2) Is it possible to put the `*` for both items a few lines before (DICOM archive and Imaging Browser)? as this note pertains to both of them.

File: docs/01-Introduction.md [hunk 32]
```diff
@@ -1,51 +1,64 @@
...
+ - the Imaging Browser module for MINC files (generated from DICOM) that pass the 
+study-defined MRI protocol 
+ - BrainBrowser using 3D or 4D navigation of these MINC files. More details on 
+BrainBrowser's capabilities can be found [here.][5]
 
+`*` Please note that all acquisitions are included in the DICOM archival
+step (even the ones excluded at a further step of the pipeline). Specific
+acquisitions (such as localizers or scouts) can be excluded from the steps of
+the pipeline that start at, and follow the DICOM to MINC conversion and the
+rest of the pipeline by specifying them in the `excluded_series_description`
```
Comment (MounaSafiHarab) [inline docs/01-Introduction.md]: can you please add this new configuration setting to section 2.2.2?

**Comment by MounaSafiHarab** [comment]
@cmadjar 
looks good, I gave you 2-3 comments. Let me know if they make sense, then I think it is ready to go :)

**Comment by cmadjar** [comment]
@MounaSafiHarab Thank you for the feedback! Let me know if I missed something.

**Comment by MounaSafiHarab** [comment]
> @MounaSafiHarab Thank you for the feedback! Let me know if I missed something.

all good on my end! I think this is ready to be merged. If you are Okay with everything, could you please squash and merge?

**Comment by cmadjar** [comment]
@MounaSafiHarab All good and merged! Thanks.
